apps/etterlatte-node-server/src/mock/mock-user.ts

```diff
@@ -1,10 +1,10 @@
 const personalia = {
-    adresse: 'Adresse-mock',
+    adresse: 'Humlegården',
```

Comment:
Hvorfor gå vekk fra mock i datafeltene?

Reply:
Tenkte bare å ha en faktisk addresse å se på for testingen sin skyld. Men er ikke big deal å gå tilbake

Reply:
Nå kommer jeg på at dette faktisk vil ovverride for de 2 andre søknaddene også 😬